app/src/main/java/org/mozilla/tv/firefox/webrender/YouTubeBackHandler.kt

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.webrender
+
+import android.view.KeyEvent
+import android.view.ViewGroup
+import android.webkit.ValueCallback
+import android.webkit.WebView
+import mozilla.components.concept.engine.EngineView
+import org.mozilla.tv.firefox.MainActivity
+import org.mozilla.tv.firefox.ext.evalJS
+import org.mozilla.tv.firefox.ext.toList
+import org.mozilla.tv.firefox.utils.URLs
+
+/**
+ * youtube/tv does not handle their back stack correctly. Going back in history visits redirects which do not alter the UI.
```

Comment:
Nit: this line is pretty long.  I don't think we have a formal max line length, but I believe most style guides suggest wrapping somewhere around 80-100 characters.  Could we do that here?